devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -981,6 +988,8 @@ void YarpRobotLoggerDevice::unpackIMU(Eigen::Ref<const analog_sensor_t> signal,
 
 void YarpRobotLoggerDevice::lookForExogenousSignals()
 {
+    std::cout << "Looking for exogenous signals" << std::endl;
```

Comment:
Leftover?

Reply:
Fixed in [d92f7f4](https://github.com/ami-iit/bipedal-locomotion-framework/pull/796/commits/d92f7f4392f6d2c5b42a7494948cb755fe16bf80)